docs/user-guide/database-configuration.md

```diff
@@ -2,7 +2,7 @@
 
 #### Postgres Version
 
-The default Postgres version for the version of AWX bundled with the latest version of the awx-operator is Postgres 13. You can find this default for a given version by at the default value for [_postgres_image_version](./roles/installer/defaults/main.yml#L138).
+The default Postgres version for the version of AWX bundled with the latest version of the awx-operator is Postgres 13. You can find this default for a given version by at the default value for [_postgres_image_version](https://github.com/ansible/awx-operator/blob/devel/roles/installer/defaults/main.yml#L243).
```

Comment:
Should we still do the relative path here?  or is the a reason for the full URL?

Reply:
When I switch it to relative, I get the following warning from `mkdocs build` : 
`Doc file 'user-guide/database-configuration.md' contains a relative link
           '../../roles/installer/defaults/main.yml#L243', but the target '../roles/installer/defaults/main.yml' is not found
           among documentation files`

Reply:
oh bummer, it's a mkdocs thing.  Markdown resolves it afaik.  ok, thx.